src/interfaces/node.h

```diff
@@ -23,7 +23,7 @@
 class BanMan;
 class CCoinControl;
 class CFeeRate;
-class CNodeStats;
+struct CNodeStats;
```

Comment:
~0 on the change from a class to a struct, I think calling it a struct is more consistent with how it's used but it doesn't seem to be a logical part of this change

Reply:
Removed the struct change.